### PR TITLE
Symfony ВС

### PR DESCRIPTION
**Pull request by kyrylich**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+.idea
```
Comment (yamilovs) [inline .gitignore]: It's not necessary.
if u want to disable some files from the stage u must to create user specific `.gitignore` file and added it to ur git config. 

For a sample like this:
```
# ~/.gitconfig

[core]
	excludesfile = /home/user/.gitignore
```
```
# /home/user/.gitignore
/.idea/
```

**Comment by yamilovs** [comment]
Hello!
Thank u for ur PR.

All is OK except one issue with gitignore file modification. Please fix it and i can merge it.

**Comment by kyrylich** [comment]
@yamilovs Hello, I removed commit with .gitignore changes

**Comment by yamilovs** [comment]
Ok, all is fine now. Thanks!

**Pull request merged**